zebrad/src/commands/start/sync.rs

```diff
@@ -77,61 +92,75 @@ where
         // due to protocol limitations
         self.request_genesis().await?;
 
-        loop {
-            self.obtain_tips().await?;
+        'sync: loop {
```

Comment:
oh damn, we getting serious over here

Reply:
When I saw this I thought "oh no, goto".

Reply:
😅